lkft/reportconfig.py

```diff
@@ -14,44 +14,7 @@
     'EAP-android12-510-db845c-presubmit': [
             '5.10-gki-private-android12-db845c-presubmit',
             ],
-    '4.4':[
-            '4.4p-10.0-gsi-hikey',
-            '4.4p-9.0-hikey',
-            ],
-    '4.9':[
-            '4.9q-10.0-gsi-hikey960',
-            '4.9q-10.0-gsi-hikey',
-            '4.9q-android11-hikey960',
```

Comment:
-q  also?

Reply:
oh can you please update the changelog?
I just thought we dropped the 4.4 and -p kernel reports.
Probably since I don't understand what ", and all the reports are done based on the kernel branches now." means...

Reply:
make sense, add more detail into the changelog.

Reply:
updated the commit message.